dist/pathfora.js

```diff
@@ -143,6 +143,7 @@
   // FUTURE Move to separate files and concat
   /* eslint-disable indent */
   var templates = {
+  'templates': {},
```

Comment:
It's strange. I've noticed that when I run the gulp tasks that gets added. It might be an extra lurking folder on my local machine or something, will investigate.